go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 replace github.com/caddyserver/caddy/v2 => ../caddyserver/caddy
```

Comment:
This is the fix

```suggestion
```

Reply:
Thanks for the hint and your quick response. I added two commits which in total only change what you requested.

Reply:
Great. Can you please run `go mod tidy`? There are changes to `go.sum` to be applied by the `go mod tidy` run